authalligator_client/entities.py

```diff
@@ -166,12 +166,34 @@ class Account(BaseAAEntity):
 
 
 @attr.attrs(frozen=True)
-class DeleteOtherAccountKeysPayload(BaseAAEntity):
+class DeleteOperation(BaseAAEntity):
+    """Base class for delete operation payloads.
+
+    These payloads don't actually have any field information in them. While
+    there's technically a "_" field in the schema, it's only a placeholder to
+    work around the language not supporting empty responses. It has no meaning
+    and will never have a meaningful value.
+
+    This class has no specific equivalent type, it's just a convenience type
+    for these entities.
+    """
+
+    pass
```

Comment:
`pass` is not needed if you have a docstring, unless you prefer to do this to explicitly show that there's nothing here.

Reply:
I do generally like the explicit `pass`, though it's really just personal preference. Class definitions without any statements just looks weird to me.